Menu/Menu.js

```diff
@@ -1,10 +1,31 @@
 
-const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+// const toggleMenu = () => {
+//   // Toggle the "menu--open" class on your menu refence. 
+// }
+
 
-// Start Here: Create a reference to the ".menu" class
-const menu;
-// create a reference to the ".menu-button" class
+// 1.. Start Here: Create a reference to the ".menu" class
+const menu = document.querySelector('.menu');
+// 2..create a reference to the ".menu-button" class
 const menuButton;
+menuButton.forEach(function(menu){
+  return new Menu(menu);
 // Using your menuButton reference, add a click handler that calls toggleMenu
+class Menu {
+  constructor(menu) {
+    this.menu = menu;
+    this.menuButtons = this.panel.querySelector('.menu-button');
+    this.menuBtnOpen = this.panel.querySelector('.expandButton');
+    //this.menuBtnClose = this.panel.querySelector('hide button???');
+    this.menuContent = this.panel.querySelector('.article');
+
+  this.menuButtons.addEventListener('hover', () => this.toggleMenu());
+  }
+
+  toggleMenu() {
+    this.menuBtnOpen.classList.toggle('expandButton');
+    this.menuBtnOpen.classList.toggle('expandButton');
+  }
```

Comment:
this is ultimately what you wanted to get 

`const toggleMenu = () => {
  menu.classList.toggle('menu--open')
}`

`const menu = document.querySelector(".menu")`
  `const menuButton = document.querySelector(".menu-button")`
  `menuButton.addEventListener('click', toggleMenu);`

we save variables that are our menu div and our menu button. then we apply an event handler to our button that fires our function toggleMenu. then in toggle menu we toggle the class on our menu